src/minitrace.c

```diff
@@ -0,0 +1,381 @@
+// minitrace
```

Comment:
`src/minitrace.c` and `src/minitrace.h` should be added to `uncrustify_exclude_files` in `dev.mk.in`.